app/javascript/widget/components/template/IntegrationCard.vue

```diff
@@ -0,0 +1,118 @@
+<template>
```

Comment:
This component is repeated and is similar to the dashboard component, however, the issues with the styles and missing shared components make it hard to build a common component.

Reply:
This piece suggests that there is an absolute need for standardization between the dashboard and widget components.